internal/datastore/spanner/watch.go

```diff
@@ -19,8 +23,21 @@ import (
 
 const (
 	RelationTupleChangeStreamName = "relation_tuple_stream"
+	SchemaChangeStreamName        = "schema_change_stream"
 )
 
+var retryHistogram = prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Why not have a single metric if both crdb and cockroach are using this?

Reply:
We could, but then it has to be defined in a central place, and that doesn't currently impact the other datastores

Reply:
Defining it in a central places seems pretty minor compared to users having to dashboard two different metrics.